src/scripts/modules/media/body/content-common.less

```diff
@@ -1,3 +1,4 @@
+@table-cell-padding: 1em;
 // Styles for the contents of a page.
```

Comment:
@PiotrKozlowski can we move this variable to content-baked.less, I am worried about regressions if we keep it in common.